.gitlab-ci.yml

```diff
@@ -8,11 +8,14 @@ include:
     file: '.gitlab-ci-check-license.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-python3-format.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-github-status-updates.yml'
 
 test:static:
   stage: test
-  image: python:3.6-buster
+  image: python:3.7-buster
   before_script:
+    - apt-get update && apt-get install -y libgirepository1.0-dev
```

Comment:
Why this was not a dependency before? I originally had this as a dependency, but then figured out it worked without it.

If it is required, we need to update the README file too

Reply:
It's failing horribly without it: https://gitlab.com/Northern.tech/Mender/mender-client-python-example/-/jobs/1372775764

Reply:
https://github.com/mendersoftware/mender-client-python-example/pull/4/commits/db43cf16d6cfb1af3d6bf5b604f952a8edb6a460

Reply:
At the beginning of my development, it failed for me the same way when working on Docker, so I added it to the README and install reqs. Then testing on Raspbian OS the introspection worked without it (did I check for the wrong package?) so I removed it for simplicity.

Anyway, no worth thinking about what I did wrong in the past. Thanks for catching!